Sources/Views/MessageInputBar.swift

```diff
@@ -378,6 +382,9 @@ open class MessageInputBar: UIView {
                     $0.parentStackViewPosition = position
                     leftStackView.addArrangedSubview($0)
                 }
+                guard let _ = superview else {
```

Comment:
@nathantannar4 , nope, I think it's needed because It can have no superview when run into `setNewItems` method. you can reproduce it when try to comment the `guard statements` .